Random card deck Complete

J-Bird22 opened the pull request:
Everything In The Code Should Be Functional Except The Deletion Of Cards Which Requires The Real New Turn Code Before It Can Be Added.

J-Bird22 committed:
Made A Basic "Block" Card

Made The HeavyShield Block Card

J-Bird22 committed:
Starting To Organize A Way To Make A Deck

J-Bird22 committed:
PickRandomCard

A basic method to pick a random but the cards are not fully implemented yet

J-Bird22 committed:
MadeAFunctioningDeck

Using The "W" Key (temporary newTurn) You can spawn in cards of Only a certain amount (ownedCards) and won't spawn anymore if you got them all already and when you run out it "Shuffles" The Decl

J-Bird22 committed:
BuggyCompleteDeck

Deck Functions But Bugs Out After 2nd Turn (Probably because off the 'i' Array)

J-Bird22 committed:
DeckFinished

The Deck Is Finished Except That The Unused Cards Need To Delete Every Turn

J-Bird22 committed:
DeckIsCompleteExceptDeletion

Deck Fully Works, Cards Get Spawned Randomly And Everything Except The Method To Delete The Cards On A New Turn Which I Need The Actual New Turn Code For.

J-Bird22 committed:
MadeAnimatedEnergyBar

Change The Energy To A New More Detailed EnergyBar

Chapmaan commented:
you've got a couple conflicts that can be resolved easily however I'm not sure which you'd like to keep for what's going in the game. other than that it looks ready to go.

J-Bird22 committed:
FixedName

Changes Name From BattleScreen To Battlescreen.

J-Bird22 committed:
FixedConflicts

Fixed A Bunch Of Pointless Conflicts
(They Were just there because greenfoot wasnt working properly)
